client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -125,22 +147,20 @@ const EditProfile: FC<Props> = (props) => {
             email: Yup.string().required('Email is required').email('Email is not valid'),
             description: loggedInUser?.isDogSitter ? Yup.string().required('Description is required.') : Yup.string(),
             city: Yup.string().required('City is required.'),
-            startDate: loggedInUser?.isDogSitter ? Yup.string().required('Start Date is required.') : Yup.string(),
-            endDate: loggedInUser?.isDogSitter ? Yup.string().required('End Date is required.') : Yup.string(),
             phone: Yup.string()
               .required('Please, enter your phone number')
               .matches(/^[0-9]+$/, 'Must contain numbers only')
               .length(10, 'Number must be 10 digits'),
             priceRate: loggedInUser?.isDogSitter
               ? Yup.string()
-                  .required('Please, enter your phone number')
+                  .required('Please, enter your Price Rate')
                   .matches(/^[0-9]+$/, 'Must contain numbers only')
                   .max(3, 'Value must br less than $100')

```

Comment:
```suggestion
                  .max(3, 'Value must be less than $100')

```

Reply:
Also, wouldn't this still allow $300 or $999

Reply:
Yes, I change max to 2.